src/components/AddItem.js

```diff
@@ -34,7 +34,7 @@ const AddItem = ({
         </Text>
         <TextInput
           onChangeText={(value) => setPrice(value)}
-          value={price}
+          value={`${price}`}
```

Comment:
I changed it to the string form because for some reason Jest was telling me that it should've been a string